airbyte-cdk/python/airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -530,7 +530,7 @@ class OffsetIncrement(BaseModel):
 
 class PageIncrement(BaseModel):
     type: Literal['PageIncrement']
-    page_size: Optional[int] = Field(
+    page_size: Optional[Union[int, str]] = Field(
```

Comment:
This file should be autogenerated with `./gradlew :airbyte-cdk:python:build`.

If you go with the suggestion above, let's run gradle to make sure the example in the model also refreshes.

Reply:
Just to make sure I'm meeting the expectations, is this a thing I should do locally and push an update? Or will this happen as part of your pairing with @marcosmarxm ?

Reply:
you can do this locally and push the change:
1. `cd airbyte-cdk/python`
2. `./gradlew --no-daemon --no-build-cache build`

Reply:
I tried running it as `./gradlew --no-daemon --no-build-cache build` from the `airbyte-cdk/python` directory, and neither has seemed to actually make any changes to this file. We're sure this is supposed to get regenerated? I even tried making trivial changes to the file, running the `build` again, and confirmed that it didn't get updated by the `gradle` run.

Reply:
Poking around a bit, seems like the `generate` task is the one we want? It did update this file in the important ways (including the type of `page_size`), _but_ it also `s/"/'/`, so I'm not sure that's what's intended?

Reply:
@girarda / @marcosmarxm - forgot to tag y'all, but if you could please advise, I can get this all ready for re-review.

Reply:
Saw the changes in `master` that updated this file per the `generateComponentManifestClassFiles` task, and I've re-run the generation on this branch.